server/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "This respository consists all server side code",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Can we have a script for start too? nodemon will be very useful for development.

Reply:
I have done it.